plugins/media-keys/msd-media-keys-manager.c

```diff
@@ -143,23 +152,23 @@ acme_error (char * msg)
 static char *
 get_term_command (MsdMediaKeysManager *manager)
 {
-	char *cmd_term, *cmd_args;
-	char *cmd = NULL;
-	GSettings *settings;
+        char *cmd_term, *cmd_args;
+        char *cmd = NULL;
+        GSettings *settings;
 
-	settings = g_settings_new ("org.mate.applications-terminal");
-	cmd_term = g_settings_get_string (settings, "exec");
-	cmd_args = g_settings_get_string (settings, "exec-arg");
+        settings = g_settings_new ("org.mate.applications-terminal");
+        cmd_term = g_settings_get_string (settings, "exec");
+        cmd_args = g_settings_get_string (settings, "exec-arg");
 
-	if (cmd_term[0] != '\0') {
-		cmd = g_strdup_printf ("%s %s -e", cmd_term, cmd_args);
-	} else {
-		cmd = g_strdup_printf ("mate-terminal -e");
-	}
+        if (cmd_term[0] != '\0') {
+                cmd = g_strdup_printf ("%s %s -e", cmd_term, cmd_args);
+        } else {
+                cmd = g_strdup_printf ("mate-terminal -e");
+        }
 
-	g_free (cmd_args);
-	g_free (cmd_term);
-	g_object_unref (settings);
+        g_free (cmd_args);
+        g_free (cmd_term);
+        g_object_unref (settings);
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.